docs/flush-tables-with-read-lock.md

```diff
@@ -72,9 +73,7 @@ running queries will be killed. The default value is zero, which turns this
 feature off.
 
 The `--kill-long-query-type` option can be used to specify all or only
-`SELECT` queries that are preventing global lock from being acquired. In order
-to use this option xtrabackup user should have `PROCESS` and `SUPER`
-privileges.
+`SELECT` queries that are preventing global lock from being acquired. To use this option xtrabackup user should have `PROCESS` and `SUPER` privileges.
```

Comment:
@altmannmarcelo can you please test if CONNECTION_ADMIN privilege is sufficient instead of SUPER to kill a query?

Reply:
https://dev.mysql.com/doc/refman/8.0/en/kill.html

>The ability to kill threads and statements depends on the [CONNECTION_ADMIN](https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_connection-admin) privilege and the deprecated [SUPER](https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_super) privilege:
> *   Without [CONNECTION_ADMIN](https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_connection-admin) or [SUPER](https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_super), you can kill only your own threads and statements.
 > *  With [CONNECTION_ADMIN](https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_connection-admin) or [SUPER](https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_super), you can kill all threads and statements, except that to affect a thread or statement that is executing with the [SYSTEM_USER](https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_system-user) privilege, your own session must additionally have the [SYSTEM_USER](https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_system-user) privilege.